fail2ban/server/database.py

```diff
@@ -34,6 +34,14 @@
 from .ticket import FailTicket
 from ..helpers import getLogger
 
+try:
+	import sqlitebck
```

Comment:
in general we have managed to avoid depending on anything but stock python and only few core libraries... this one seems to be quite concise, but not even present in debian and seems might have issues on osx (https://github.com/husio/python-sqlite3-backup/issues/5)... so to jump that way it would be nice to have concrete numbers of how beneficial this endeavor would be